post-scan-actions/aws-python-email-notification/README.md

```diff
@@ -0,0 +1,262 @@
+    - `<YOUR_ZIP_NAME>` is replaced with the name of the ZIP file you created earlier. Example: `scan-send-email`
+    - `<YOUR_REGION>` is replaced by the region where the scanning bucket does reside in
+    - `<YOUR_SENDER>` is replaced with the name of your sender Email address.
+    - `<YOUR_RECIPIENT>` is replaced with the name of your recipient Email address.
+    - `<YOUR_SUBJECT>`is replaced with the subject of your Email.
+</details>
+
+<details>
+<summary>Using the Makefile</summary>
+
+1. Download the [Makefile from GitHub](https://github.com/trendmicro/cloudone-filestorage-plugins/blob/master/post-scan-actions/aws-python-email-notification/Makefile).
+2. In a shell program, enter the following GNU Make command, using backslashes (`\`) to separate lines, as shown below:
+
+    ```bash
+    FUNCTION_NAME=<YOUR_FSS_SCAN_SEND_EMAIL> ROLE_ARN=<YOUR_FSS_LAMBDA_EMAIL_ROLE> \
+    SENDER=<YOUR_SENDER> RECIPIENT=<YOUR_RECIPIENT> SUBJECT=<YOUR_SUBJECT> \
+    make create-function
+    ```
+- where:
+    - `<YOUR_FSS_SCAN_SEND_EMAIL>` is replaced with the name you want to give your Lambda function. Example: `FSS_Scan_Send_Email`.
+    - `<YOUR_FSS_LAMBDA_EMAIL_ROLE>` is replaced with the ARN of the role you previously created for the Lambda function. You can find the ARN in the AWS console under **Services > IAM > Roles** > your role > **Role ARN** field (at the top). Example: `arn:aws:iam::012345678901:role/FSS_Lambda_Email_Role`.
+    - `<YOUR_SENDER>` is replaced with the name of your sender Email address.
+    - `<YOUR_RECIPIENT>` is replaced with the name of your recipient Email address.
+    - `<YOUR_SUBJECT>`is replaced with the subject of your Email.
+</details>
+
+## Subscribe the Lambda to the SNS topic
+
+<details>
+<summary>Using the AWS console</summary>
+
+1. Go to **Services > Lambda**.
+2. Search for the Lambda function you created previously. Example: `FSS_Scan_Send_Email`
+3. Click the link to your Lambda function to view its details.
+4. Click **Add trigger** on the left.
+5. From the **Trigger configuration** list, select **SNS**.
+6. In the **SNS topic** field, enter the SNS topic ARN you found earlier.
+7. Click **Add**. Your Lambda is now subscribed to the SNS topic.
+
+</details>
+
+<details>
+<summary>Using the AWS CLI</summary>
+
+1. **Find the 'ScanResultTopic' SNS topic ARN** 
+    - In the AWS console, go to **Services > CloudFormation** > your all-in-one stack > **Resources** > your storage stack > **Resources**.
+    - Scroll down to locate the  **ScanResultTopic** Logical ID. 
+    - Copy the **ScanResultTopic** ARN to a temporary location. Example: `arn:aws:sns:us-east-1:123445678901:FileStorageSecurity-All-In-One-Stack-StorageStack-1IDPU1PZ2W5RN-ScanResultTopic-N8DD2JH1GRKF`
+2. **Find the Lambda function ARN**
+    
+    📌 The Lamdba function ARN is only required if you plan to use the AWS CLI (as opposed to the console) to subscribe the Lambda to the SNS topic.
```

Comment:
📌 The Lamdba function ARN is required only if you plan to use the AWS CLI (as opposed to the console) to subscribe the Lambda to the SNS topic.

Reply:
Ok

Reply:
Changed